1.7/administration/installing/custom/cli.md

```diff
@@ -310,9 +310,14 @@ To install DC/OS:
     # <Ctrl-C> to exit installer
     $ cd genconf/serve
     $ sudo tar cf dcos-install.tar *
-    # Move the dcos-install.tar file to a safe place
+
```

Comment:
I'd like to see some more text here around *why* this is being backed up. The stuff from the GUI installer is probably good enough.